docs/OnlineTutorial/Sequences.md

```diff
@@ -36,8 +36,8 @@ sorted, then the whole array is sorted:
 ``` {.edit}
 predicate sorted2(s: seq<int>)
 {
-   0 < |s| ==> (forall i :: 0 < i < |s| ==> s[0] <= s[i]) &&
-               sorted2(s[1..])
+  0 < |s| ==> (forall i :: 0 < i < |s| ==> s[0] <= s[i]) &&
+              sorted2(s[1..])
```

Comment:
```suggestion
  0 < |s| ==> (forall i :: 0 < i < |s| ==> s[0] <= s[i]) &&
               sorted2(s[1..])
```

Reply:
I actually prefer to align it this way because of the opening parenthesis that defines a block. Hence, visually, it's easier to see that the entire parenthese is a block.
One could break this block if there is a newline after the parenthesis in the new formatter, e.g.

```
 0 < |s| ==> (
   forall i :: 0 < i < |s| ==> s[0] <= s[i]) &&
   sorted2(s[1..])
```

Reply:
+1 for auto indentation. I missed the `)` that closed the forall. So I agree with your indenting on this one.